src/components/Forms/NewEvent.js

```diff
@@ -8,11 +8,15 @@ import {
     KeyboardDateTimePicker
 } from '@material-ui/pickers';
 import ThemeProvider from '../ThemeProvider'
+import InfoIcon from '@material-ui/icons/Info'
+import Select from "@material-ui/core/Select";
+import MenuItem from '@material-ui/core/MenuItem';
+import { Typography } from '@material-ui/core';
```

Comment:
```suggestion
import Typography from '@material-ui/core/Typography';
```

will allow for faster builds